### ISSUE TITLE
When i download a video from the playlist, it downloads the whole videos in the playlist

### ISSUE THREAD

**Issue by chessssssssss** [jely2002/youtube-dl-gui#348]
**Describe the bug**
When im trying to download a video from a playlist like a meme playlist. it downloads the whole videos in the playlist.

**To Reproduce**

**Expected behavior**
I didnt expect to download the whole videos in the playlist

**Additional info (please complete the following information):**
 - OS: Windows 10
 - Application version: 2.4.0
 - Application type: installer/real app (not portable or the microsoft app store)

**Comment by olegshulyakov** [comment]
@chessssssssss Can you please add more details?
To download single video from playlist, all you need is to click download button for selected video file:
<img width="1152" alt="image" src="https://user-images.githubusercontent.com/4524455/174449281-a21b0f9f-f8cc-405b-b966-3c1870523825.png">

**Comment by jely2002** [comment]
In the settings of v3 you can select whether a URL with both a playlist ID and a video ID in it, will result in downloading the playlist or the video:

<img width="645" height="277" alt="Image" src="https://github.com/user-attachments/assets/c21a4c22-8820-4654-a532-4633b698462c" />

By default we mirror what ytdlp does, which is downloading the entire playlist, instead of just the one video. 

TL;DR in version 3 of the app you can use the setting to decide what to download.
